### ISSUE TITLE
The "backups.full_backup" action fails.

### ISSUE THREAD

**Issue by stevenmcastano** [StackStorm/st2#4695]
The "backups.full_backup" action fails. At first it was complaining that there was note role "ROOT", so I manually created one and gave it superuser permissions, but not it appears to be able to log in, however it's hanging and the execution never finishes. The only thing I can see is the following in the postgres log:

```
2019-05-28 13:28:03.308 EDT [46825] root@postgresql+psycopg2://mistral:StackStorm@localhost/mistral FATAL:  database "postgresql+psycopg2://mistral:StackStorm@localhost/mistral" does not exist
```

##### STACKSTORM VERSION
```
st2 3.0.0, on Python 2.7.15
```

##### OS / ENVIRONMENT / INSTALL METHOD
Ubuntu 18.04 installed manually

**Comment by LindsayHill** [comment]
If you're using Ubuntu 18, then you're using our unstable packages, and there is no st2mistral component, and hence no need for a PostgreSQL DB. 

You would only need to backup the Mongo DB.

**Comment by LindsayHill** [comment]
Also - if you have issues, bugs or feature requests for the backups pack, you can post them against that pack's repo - see github.com/StackStorm-Exchange/stackstorm-backups

**Comment by blag** [comment]
https://github.com/StackStorm-Exchange/stackstorm-backups

**Comment by nmaludy** [comment]
FYI, you can just run `backups.mongodb_backup` instead of `backups.full_backup`

**Comment by stevenmcastano** [comment]
I did install everything from the xenial packages though... so postgres 10 is running, the mistral database did get initialized and I do see that there is data in it. I was just trying to install it on a more up-to-date OS. 16.04 is over 3 years old now.

**Comment by LindsayHill** [comment]
18.04 is not Xenial - those packages (and install instructions) apply to Ubuntu 16.04, not 18.04.

See https://docs.stackstorm.com/install/system_requirements.html for currently supported systems.

Ubuntu 18.04 packages are in our unstable repo. In future they will promoted to GA, and install instructions and system requirements will be updated.

> 16.04 is over 3 years old now.

We couldn't start adding 18.04 support until MongoDB 4.0 was released. ST2 on Ubuntu 18.04 will only support Python3, which required additional testing.

**Comment by stevenmcastano** [comment]
I understand xenial is not 18.04... that's what I was trying to say... I was installing the Xenial package on 18.04 (bionic) because I wanted to stay with stable packages... I did read the system requirements, I was just trying to run it on a newer OS that would have much longer LTS support, then just be able to upgrade StackStorm once the 18.04 packages are stable.

I'm going to try the install again on 16.04... in fact I've already started. I was just hoping to avoid using an older OS that will be end of like in April 2021... that's only 2 years away and I like to build systems for our datacenters that I know I can count on through the lifecycle of the environment and it's hardware, 3 - 4 years.
